docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -85,6 +56,34 @@ lagom.persistence.read-side.cassandra {
 }
 ```
 
-This setup disables Lagom's `ConfigSessionProvider` and falls back to that provided in `akka-persistence-cassandra` which uses the list of endpoints listed in `contact-points`.
+## Using static values for services and Cassandra to simulate a managed runtime 
+
+While we would never advise using static service locations in production, to simulate a working Lagom system in the absence of a managed runtime, you can deploy Lagom systems to static locations by using static configuration. When using static service location, you can also hardcode Cassandra locations. To achieve this, you will need to:
+
+1. Specify service locations in `application.conf`.
+2. Bind the `ConfigurationServiceLocator` in your Guice module.
+3. Optionally add Cassandra locations in `application.conf`.
+
+The  [`ConfigurationServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/client/ConfigurationServiceLocator.html) reads the service locator configuration out of Lagom's `application.conf` file.  Here is an example that specifies static locations for two Lagom services:
+
+```
+lagom.services {
+  serviceA = "http://10.1.2.3:8080"
+  serviceB = "http://10.1.2.4:8080"
+}
+```
+
+To instruct Lagom to use the `ConfigurationServiceLocator`, you need to bind it to the [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) class in your Guice module. Of course, you don't have to configure a separate module, this configuration can be added to your existing Guice module. Since Lagom already provides a service locator in dev mode, you will likely only want to bind this when Lagom is in production mode.  Play supports passing its `Environment` and `Configuration` objects to module constructors, so you'll need to add those to your module:
+
+@[content](code/docs/production/ConfigurationServiceLocatorModule.java)
+
+
+To hardcode the Cassandra contact points when using a static service locator, add the following to the `application.conf` file:
+
+```
+lagom.services {
+  cas_native = "tcp://10.1.2.3:9042"
+}
+```
+
 
```

Comment:
I agree that the info wrt cassandra setup looks a bit out of place. I think we can tackle that in a separate PR though.

In that separate PR I think we could improve the Deployment considerations: package, service location, kafka endpoints, cassandra, rdbms, etc... I think we're lacking some information here. But I'd expedite this PR to ealy publicise support for k8s.